tests/matrix_free/point_evaluation_14.output

```diff
@@ -2,291 +2,291 @@
 DEAL::Mapping of degree 2
 DEAL::Cell with center 0.6495190528 0.3750000000
 DEAL::0.9921875000 0.000000000: 1.000000000 error value 0.000000000 error grad 0.000000000
-DEAL::0.9591965610 0.06332871922: 1.063328719 error value -2.220446049e-16 error grad 5.311767473e-16
-DEAL::0.9234265460 0.1210574346: 1.121057435 error value 0.000000000 error grad 6.206335383e-16
-DEAL::0.8851608469 0.1733497625: 1.173349762 error value -2.220446049e-16 error grad 1.696726283e-15
-DEAL::0.8446828558 0.2203693193: 1.220369319 error value 4.440892099e-16 error grad 8.005932085e-16
-DEAL::0.8022759646 0.2622797217: 1.262279722 error value 2.220446049e-16 error grad 3.885780586e-16
-DEAL::0.7582235654 0.2992445859: 1.299244586 error value -2.220446049e-16 error grad 2.220446049e-16
-DEAL::0.7128090500 0.3314275284: 1.331427528 error value 2.220446049e-16 error grad 1.276756478e-15
-DEAL::0.6663158106 0.3589921657: 1.358992166 error value 0.000000000 error grad 4.577566799e-16
-DEAL::0.6190272391 0.3821021142: 1.382102114 error value 0.000000000 error grad 3.376611507e-16
-DEAL::0.5712267275 0.4009209903: 1.400920990 error value 0.000000000 error grad 5.551115123e-16
-DEAL::0.5231976677 0.4156124105: 1.415612411 error value 2.220446049e-16 error grad 3.376611507e-16
-DEAL::0.4752234518 0.4263399913: 1.426339991 error value 0.000000000 error grad 1.093442870e-15
+DEAL::0.9591965610 0.06332871922: 1.063328719 error value 0.000000000 error grad 0.000000000
+DEAL::0.9234265460 0.1210574346: 1.121057435 error value 0.000000000 error grad 0.000000000
+DEAL::0.8851608469 0.1733497625: 1.173349762 error value 0.000000000 error grad 0.000000000
+DEAL::0.8446828558 0.2203693193: 1.220369319 error value 0.000000000 error grad 0.000000000
+DEAL::0.8022759646 0.2622797217: 1.262279722 error value 0.000000000 error grad 0.000000000
+DEAL::0.7582235654 0.2992445859: 1.299244586 error value 0.000000000 error grad 0.000000000
+DEAL::0.7128090500 0.3314275284: 1.331427528 error value 0.000000000 error grad 0.000000000
+DEAL::0.6663158106 0.3589921657: 1.358992166 error value 0.000000000 error grad 0.000000000
+DEAL::0.6190272391 0.3821021142: 1.382102114 error value 0.000000000 error grad 0.000000000
+DEAL::0.5712267275 0.4009209903: 1.400920990 error value 0.000000000 error grad 0.000000000
+DEAL::0.5231976677 0.4156124105: 1.415612411 error value 0.000000000 error grad 0.000000000
+DEAL::0.4752234518 0.4263399913: 1.426339991 error value 0.000000000 error grad 0.000000000
 DEAL::
-DEAL::-3.748628693 0.000000000 -3.173305599 0.000000000 1.519866565 0.000000000 0.3917604893 0.000000000 -15.07404237 0.000000000 9.522955246 0.000000000 20.12140751 0.000000000 -10.85961324 0.000000000 17.75462483 0.000000000 
+DEAL::0.000000000 -3.748628693 0.000000000 -3.173305599 0.000000000 1.519866565 0.000000000 0.3917604893 0.000000000 -15.07404237 0.000000000 9.522955246 0.000000000 20.12140751 0.000000000 -10.85961324 0.000000000 17.75462483 
```

Comment:
This and other lines are computed with `first_component=1` so the results should be shifted by one. Therefore, fixing the renumbering also fixes this test.